roles/common/meta/main.yml

```diff
@@ -14,3 +14,6 @@ dependencies:
     when: docker.enabled is defined and docker.enabled|bool
   - role: monitoring-common
     when: monitoring.enabled is not defined or monitoring.enabled|bool
+  - role: logging-config
+    service: auditd
```

Comment:
We likely want this to actually be `service: common` instead, as the logdata is defined in `common` and we currently try to only have 1 logdata definition per role.